src/pytorch_lightning/accelerators/__init__.py

```diff
@@ -12,7 +12,7 @@
 # limitations under the License.
 from pytorch_lightning.accelerators.accelerator import Accelerator  # noqa: F401
 from pytorch_lightning.accelerators.cpu import CPUAccelerator  # noqa: F401
-from pytorch_lightning.accelerators.gpu import GPUAccelerator  # noqa: F401
+from pytorch_lightning.accelerators.cuda import CUDAAccelerator  # noqa: F401
```

Comment:
this breaks
```py
from pytorch_lightning.accelerators import GPUAccelerator
```
we should keep the import

Reply:
we shouldn't because then it will automatically be registered for the gpu keyword but we want it to map to the CUDAAccelerator